.gitignore

```diff
@@ -2,6 +2,5 @@ dist/
 node_modules/
 vendor/
 .gh_token
-composer.lock
```

Comment:
`composer.lock` should be ignored here, and removed from the `.gitignore` file by the `plugin.sh` script.